mira/dkg/model.py

```diff
@@ -267,22 +275,40 @@ def get_conversion_cls(self) -> Type[Template]:
 
 @model_blueprint.post("/stratify", response_model=TemplateModel, tags=["modeling"])
 def model_stratification(
+    request: Request,
     stratification_query: StratificationQuery = Body(
         ...,
         example={
             "template_model": template_model_example,
             "key": "city",
-            "strata": ["boston", "nyc"],
+            "strata": ["geonames:4930956", "geonames:5128581"],
+            "strata_name_map": {"geonames:4930956": "Boston",
```

Comment:
here, it shouldn't be necessary to provide the name mapping since the API can interact with the DKG and look the names up

Reply:
I changed the example here to use the flag now